deployment/GP02HYPM_Deploy.csv

```diff
@@ -1,99 +1,99 @@
+TN323,,RB1605,,GP02HYPM-WFP03-05-VEL3DL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.07983,-144.803,,4070,4219,
+#TN323,,RB1605,,GP02HYPM-MPM01-02-ZPLSGA009,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,,,50.07983,-144.803,,,4220,Not Deployed
+#TN323,,RB1605,,GP02HYPM-MPM01-02-ZPLSGA010,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,,,50.07983,-144.803,,,4220,Not Deployed
+RB1605,,SR1710,,GP02HYPM-MPM01-02-ZPLSGA009,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,,CGINS-ZPLSGA-55093,50.0796,-144.80572,,150,4220,upward-looking
+RB1605,,SR1710,,GP02HYPM-MPM01-02-ZPLSGA010,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,,CGINS-ZPLSGA-55092,50.0796,-144.80572,,150,4220,downward-looking
+RB1605,,SR1710,,GP02HYPM-RIM01-00-SIOENG000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,,CGCON-SIOCON-12369,50.0796,-144.80572,,4072,4220,
+RB1605,,SR1710,,GP02HYPM-RIM01-02-CTDMOG039,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,,CGINS-CTDMOG-11642,50.0796,-144.80572,,160,4220,
+RB1605,,SR1710,,GP02HYPM-WFP02-00-WFPENG000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-10402,CGCON-GWFPGX-10402,50.0796,-144.80572,,2094,4220,
+RB1605,,SR1710,,GP02HYPM-WFP02-01-FLORDL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-10402,CGINS-FLORDL-03283,50.0796,-144.80572,,2094,4220,
+RB1605,,SR1710,,GP02HYPM-WFP02-03-DOSTAL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-10402,CGINS-DOSTAL-01480,50.0796,-144.80572,,2094,4220,
+RB1605,,SR1710,,GP02HYPM-WFP02-04-CTDPFL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-10402,CGINS-CTDPFL-00134,50.0796,-144.80572,,2094,4220,
+RB1605,,SR1710,,GP02HYPM-WFP02-05-VEL3DL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-10402,CGINS-VEL3DL-01108,50.0796,-144.80572,,2094,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-00-WFPENG000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGCON-GWFPGX-93602,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-01-FLORDL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-FLORDL-02736,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-03-DOSTAL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-DOSTAL-01086,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-04-CTDPFL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-CTDPFL-00107,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-05-VEL3DL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-VEL3DL-01024,50.0796,-144.80572,,4062,4220,

```

Comment:
Deployment 4: WFP03 and SIOENG stopDateTime should be 2017-07-16T18:24 (per mooring recovery log).

Reply:
done